### PR TITLE
Updated scaling.md for revised scaling docs

### PR DESCRIPTION
**Pull request by aashishkohli**
Updated autoscaling docs

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
This is an automated comment for commit 77be8f9fb82a756e0aa5c7ae4672eab06b023bf5 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1516/77be8f9fb82a756e0aa5c7ae4672eab06b023bf5/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

**Review by justindeguzman — APPROVED** [review]

**Comment by tbragin** [comment]
Although I was not listed as a reivewer, I took a look and it appears to be an improvement. Thank you for working on this @aashishkohli !

File: docs/en/cloud/manage/scaling.md [hunk 1]
```diff
@@ -5,35 +5,41 @@ slug: /en/manage/scaling
...
+Use automatic idling only if your use case can handle a delay before responding to queries, because when a service is paused, connections to the service will time out. Automatic idling is ideal for services that are used infrequently and where a delay can be tolerated. It is not recommended for services that power customer-facing features that are used frequently.
 :::
+
+## Handling bursty workloads
+If you have an upcoming expected spike in your workload, you can use the
+[ClickHouse Cloud API](/docs/en/cloud/manage/api/services-api-reference.md) to preemptively scale up your service to handle the spike and scale it down once the demand subsides. To understand current service size and the number of replicas you can run the query below
+
+```
+SELECT hostname(), getSetting('max_threads'), formatReadableSize(getSetting('max_memory_usage'))
```
Comment (melvynator) [inline docs/en/cloud/manage/scaling.md:43]: Should be replace by:

```
SELECT *
FROM clusterAllReplicas('default', view(
    SELECT
        hostname() AS server,
        getSetting('max_threads') as cpu_cores,
         formatReadableSize(getSetting('max_memory_usage')) as memory
    FROM system.one
))
ORDER BY server ASC
SETTINGS skip_unavailable_shards = 1
```

**Review by melvynator — CHANGES_REQUESTED** [review]

File: docs/en/cloud/manage/scaling.md (hunk 1 above)
Comment (justindeguzman) [reply docs/en/cloud/manage/scaling.md:43]: Thanks! I'll update this

**Pull request merged**